package.json

```diff
@@ -231,6 +231,7 @@
       "trim-newlines@<3.0.1": "^3.0.1",
       "xml2js@<0.5.0": "^0.5.0",
       "@types/mocha": "^10.0.8",
+      "rollup@<2.79.2": "^2.79.2",
       "rollup@>=4.0.0 <4.22.4": "^4.22.4",
```

Comment:
Why did it migrate compnents that did use roll up for to use 2.79.2? Would n't expect them to be modified in the lock file